.pre-commit-config.yaml

```diff
@@ -47,6 +47,14 @@ repos:
         args: ["--toml", "pyproject.toml"]
         exclude: (?x)^(.*stemmer.*|.*stop_words.*|^CHANGELOG.md$|.*\.md|.*\.rs$|src/utils/serdes/serdes.h)
 
+  - repo: local
+    hooks:
+      - id: staged-clang-format
+        name: clang-format
+        entry: git clang-format
+        language: system
+        files: \.(cpp|h|hpp|cc|c|cxx|cu|cuh)$
```

Comment:
is it really needed for git clang-format?
maybe pass ` pass_filenames: false` instead?